Sources/ArcGISToolkit/Components/Offline/OfflineManager.swift

```diff
@@ -17,7 +17,54 @@ import Combine
 import OSLog
 import SwiftUI
 
-/// An object that maintains state for the offline components.
+/// A utility class that maintains states of offline map areas and their
+/// file structures on the device.
+///
+/// This component utilizes the ``JobManager`` and provides high-level APIs to
+/// manage offline map areas and access their data.
```

Comment:
The first part of this sentence is unrelated to the second part of the sentence. Also, I think the JobManager aspect is secondary or tertiary to what this type does. So I would break it up into two sentences and change the order:

```suggestion
/// This component provides high-level APIs to manage offline map areas and access their data.
/// This component manages offline jobs and it utilizes the ``JobManager`` in order to facilitate jobs
/// continuing to run while the app is backgrounded.
```